More easy run (#571)
JabRef/user-documentation: en/contributing/README.md

  - Get `gg.cmd` (using either `wget` or `curl`)
    - `wget ggcmd.io/gg.cmd`
    - Alternative: `curl -L ggcmd.io > gg.cmd`

>   - Have JBang trusting JabRef's code

Reviewer: ```suggestion
  - Have JBang trust JabRef's source
```

    - `sh ./gg.cmd jbang trust add https://github.com/JabRef/jabref/`
  - Clone JabRef
    - `sh ./gg.cmd jbang https://github.com/JabRef/jabref/blob/main/.jbang/CloneJabRef.java jabref`

>     - NOTE: You can also use the native git client: `git clone --recurse-submodules https://github.com/JabRef/jabref.git` for achieving the same result.

Reviewer: ```suggestion
    - NOTE: You can also use the native git client: `git clone --recurse-submodules https://github.com/JabRef/jabref.git` to achieve the same result.
```

  - Make `gg.cmd` available in `jabref` source directory
    - `cd jabref`
    - `mv ../gg.cmd .`

Now you are all set: You have a directory `jabref` containing the recent updates and also `gg.cmd` which you will need later for executing a JabRef build.

Note: If you don't want to store JabRef's source code permanently, you can follow the steps at [our blog post on gg.cmd usage](https://blog.jabref.org/2025/05/31/run-pr/). There, JabRef's source is checked out in a temporary directory.

### Try a branch

1. `cd` into the `jabref` source directory: `cd c:\git-repositories\jabref`
2. Checkout the PR and run JabRef: `sh ./gg.cmd just run-pr <pr-number>` - replace `<pr-number>` with the PR number or the unique branch identifier by GitHub

   - Example: `13182` for [pr#13182](https://github.com/JabRef/jabref/pull/13182).

>    - Example: `Yubo-Cao:walkthroug` for the branch identifier output by GitHub\

Reviewer: ```suggestion
   - Example: `Yubo-Cao:walkthrough` for the branch identifier output by GitHub\
```

### Try a branch

1. `cd` into the `jabref` source directory: `cd c:\git-repositories\jabref`
2. Checkout the PR and run JabRef: `sh ./gg.cmd just run-pr xyz` - replace `xyz` with the PR number or the unique branch identifier by GitHub

   - Example: `13182` for [pr#13182](https://github.com/JabRef/jabref/pull/13182).
   - Example: `Yubo-Cao:walkthroug` for the branch identifier output by GitHub\
     ![pr-13182](../.gitbook/assets/pr-13182.png)

This will download the necessary JDK and a gradle distribution.
On the first run, please give the system enough time to accommodate and wait until the JabRef window launches.
Depending on your hardware, this may take a few minutes.

On Windows, instead of `sh ./gg.cmd` use `.\gg.cmd`.

#### Alternatives

1. In case you don't want to use `gg.cmd`: You can install [JBang](https://www.jbang.dev/) and a [Java JDK](https://adoptium.net/de/temurin/releases/?os=any&arch=any&version=21)for yourself and execute the commands directly. These are

>    - `jbang https://github.com/JabRef/jabref/blob/main/.jbang/CheckoutPR.java {{pr-id}}`

Reviewer: Are the double curly braces part of the argument syntax?

Author: Above, I had xyz - I replaced all with `<pr-number>`
